03-Raster-Calculations-In-R.Rmd

```diff
@@ -96,24 +109,15 @@ buildings, etc with the influence of ground elevation removed.
 <figure>
     <a href="{{ site.baseurl }}/images/raster_timeseries/lidarTree-height.png">
     <img src="{{ site.baseurl }}/images/raster_timeseries/lidarTree-height.png"></a>
-    <figcaption>A canopy height model represents the difference between the digital 
-    surface model and a digital terrain model. It represents the actual height 
-    of the trees with the influence of elevation, removed.</figcaption>
+    <figcaption> A canopy height model represents the difference between the
+    digital surface model and a digital terrain model. It represents the actual
+    height of the trees with the influence of elevation removed. Image source:
+    NEON, Inc.</figcaption>
 </figure>
 
-> * <a href="http://neondataskills.org/remote-sensing/2_LiDAR-Data-Concepts_Activity2/" target="_blank">More on LiDAR CHM, DTM and DSM here.</a>
-
-We can calculate the difference between two rasters in two different ways:
```

Comment:
I like to lay out steps for people. It engrains it in their minds the options. If it's all linear, things get lost. 

Is there any reason why you removed this?